### ISSUE TITLE
updating dataFields for many/all series at once

### ISSUE THREAD

**Issue by noxjonas** [amcharts/amcharts4#3513]
Hi,

I have buttons that should change the dataFields for one of the axes.
i.e. ``[{x: 1, y1: 0.5, y2: 0.0003},...]`` per series, where I want to change from y1 to y2

Currently I am looping over each series, changing the dataFields.valueY and calling .validateData() on each series. This, however, is extremely slow with 100 series. 

Is there a more efficient way to achieve this?

**Comment by martynasma** [comment]
> Is there a more efficient way to achieve this?

I don't think so.

How about pre-creating series for all variants of dataFields and just hiding/showing them as needed?

**Comment by noxjonas** [comment]
I have now tried this, however it is too laggy. Loading bar pops up sometimes even. I have turned animations off.

I still end up looping all the series, meaning that it changes the hidden (or disabled - seems to be better) properties one by one, which I'm not sure is helping as it is updating the plot as it goes along. Perhaps there's a way to disable rendering while this is happening, a batch update of sorts?
